.changeset/silent-years-bake.md

```diff
@@ -0,0 +1,11 @@
+---
+'@backstage/plugin-scaffolder-backend-module-rails': patch
+'@techdocs/cli': patch
+'@backstage/cli': patch
+'@backstage/cli-common': patch
+'@backstage/create-app': patch
+'@backstage/codemods': patch
+'@backstage/repo-tools': patch
+---
+
+Bumped `@types/node` dependency to `^18.17.8`.
```

Comment:
There's also the mock-fs bump here and there, is that less worth mentioning than the type bump? Not sure